rtrlib/rtr/rtr.c

```diff
@@ -240,10 +244,17 @@ void rtr_stop(struct rtr_socket *rtr_socket)
     RTR_DBG1("rtr_stop()");
     rtr_change_socket_state(rtr_socket, RTR_SHUTDOWN);
     if(rtr_socket->thread_id != 0) {
-        RTR_DBG1("pthread_kill()");
-        pthread_kill(rtr_socket->thread_id, SIGUSR1);
```

Comment:
The sig handler is now dead code, right?

Reply:
yes, removed.